target/linux/ath79/dts/qca9558_engenius_emr3000-v2.dts

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca955x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "EnGenius EMR3000 V2";
+	compatible = "engenius,emr3000-v2", "qca,qca9558";
+
+	aliases {
+		led-boot = &status_white;
+		led-failsafe = &status_white;
+		led-running = &status_white;
+		led-upgrade = &status_white;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		status_white: status_white {
```

Comment:
This should be
`led_status_white: status_white {`

Reply:
Thanks for comment.